tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -215,7 +215,7 @@
 
             # Do not attempt OVS major upgrades here
             edpm_ovs_packages:
-            - openvswitch3.1
+            - rdo-openvswitch-3.1
```

Comment:
if there are some downstream jobs running adoption jobs those needs to be adjusted too

Reply:
Right. I am not sure what the right way to distinguish between the two environments. @jistr is this something we care about for test suite?

Reply:
Yes the test suite should support both downstream and upstream. If there is a difference between the name of the package between upstream and downstream, i think we can parametrize the test suite (and likely default to the upstream value).